src/base/kaldi-math.h

```diff
@@ -183,6 +183,11 @@ inline Float RandPrune(Float post, BaseFloat prune_thresh,
 
 inline double LogAdd(double x, double y) {
   double diff;
+  if (!std::isfinite(y))
```

Comment:
I'm concerned about this.. there are other parts of the code where we call LogAdd, and these changes will make it slower.  Was this really necessary?